### PR TITLE
conf: Added support for db and api testing

### PR DESCRIPTION
**Pull request by gonzcm**
## Description
Now it is possible to create and perform tests on the code base.

## Related Issue(s)
- #27 

## Checklist
- [x] I have read and followed the project's contributing guidelines.
- [x] I have tested the changes locally.
- [x] I have made corresponding changes to the documentation if necessary.
- [x] My code follows the project's code style guidelines.

## Reviewers
- @jmartinacu 
- @alvgonfri 

## Notes

For testing, we use a different postgres database than the one we use for development. To configure this database, the uri must be changed inside the `conftest.py` file in the tests folder.

To launch the tests, just run the `pytest` command in the main directory of the project.

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+async def connection(engine: AsyncEngine):
+    async with engine.connect() as conn:
+        trans = await conn.begin()
+        yield conn
+        await trans.rollback()
+
+
+@pytest_asyncio.fixture
+async def session(connection):
+    AsyncSessionLocal = sessionmaker(
```
Comment (jmartinacu) [inline tests/conftest.py:40]: This variable doesn't follow snake case naming convention.

**Review by alvgonfri — APPROVED** [review]
Testing capability is now enabled.

**Review by jmartinacu — APPROVED** [review]
OK

**Review by alvgonfri — APPROVED** [review]
Good improvements!

**Pull request merged**